scout/server/blueprints/variants/templates/variants/utils.html

```diff
@@ -412,6 +412,7 @@
 <input type="hidden" name="variant_type" value="{{ form.variant_type.data }}">
 <div class="row mb-3">
   {{ variants_common_filters(form, "sv") }}
+</div>
```

Comment:
Oh no, it was this! 🙅🏻

Reply:
But, hey, sorry, I missed pushing the rest, didn't I! That was just the layout. The fix for the folding is the missing argument.

Reply:
It was working also before, but folding from the right instead of from the bottom, weird!

Reply:
I think it may also have been a little browser dependent. And the title bar could end up getting a fancy rounding sometimes. Anyway, that was not why it didn't save the toggle state, only why the toggle was a bit hidden away.